source/dyaml/emitter.d

```diff
@@ -73,6 +73,12 @@ private alias canFind = std.algorithm.canFind;
 //Emits YAML events into a file/stream.
 struct Emitter 
 {
+
+    // disable moves /copy b/c state_ carries original this context
+    @disable this(this);
```

Comment:
Possible breakage in user code. But without this i cant see how to trust the two methods